src/include/network/lan/hardware.rb

```diff
@@ -829,7 +828,7 @@ def storeHW(_key, _event)
         end
         if LanItems.type == "vlan"
           # for vlan devices named vlanN pre-set vlan_id to N, otherwise default to 0
-          LanItems.vlan_id = "#{nm["vlan".size].to_i}"
+          LanItems.vlan_id = nm[VLAN_SIZE].to_i.to_s
```

Comment:
ah it wasn't you who updated that ... but anyway. Could you pls fix that before it disappears in code depth again? Thanks